src/routes/email/exception-filters/email-already-verified.exception-filter.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  ArgumentsHost,
+  Catch,
+  ExceptionFilter,
+  HttpStatus,
+} from '@nestjs/common';
+import { EmailAlreadyVerifiedError } from '@/domain/email/errors/email-already-verified.error';
+import { Response } from 'express';
+
+@Catch(EmailAlreadyVerifiedError)
+export class EmailAlreadyVerifiedExceptionFilter implements ExceptionFilter {
+  catch(exception: EmailAlreadyVerifiedError, host: ArgumentsHost): any {
```

Comment:
Nit: can we change `any` to `void`? Type checking is otherwise disabled.

Reply:
The return type comes from the interface `ExceptionFilter` (NestJS).

I've removed the return type hint here though. [f0f5b82](https://github.com/safe-global/safe-client-gateway/pull/919/commits/f0f5b8208024628b4bab921e6e5d36f40e9a95fb)